src/server/pfs/server/driver_file.go

```diff
@@ -92,7 +92,7 @@ func (d *driver) withCommitUnorderedWriter(ctx context.Context, renewer *fileset
 	return d.commitStore.AddFileSet(ctx, commit, *id)
 }
 
-func (d *driver) withUnorderedWriter(ctx context.Context, renewer *fileset.Renewer, compact bool, cb func(*fileset.UnorderedWriter) error, opts ...fileset.UnorderedWriterOption) (*fileset.ID, error) {
```

Comment:
So now we just always compact.

Reply:
No, we only compact for file sets created through the file set API. Also, the compaction just keeps the file sets under the fan in, it doesn't compact into one file set.